lib/actions/form.js

```diff
@@ -37,12 +38,18 @@ export function setTime (payload) {
   }
 }
 
+let debouncedPlanTrip  // store as global, so it can be reused.  Maybe there's a better way?
```

Comment:
If `debouncePlanTimeMs` can change then you would need to recreate the debounced function when it does. If it can't change then it's application configuration and shouldn't be retrieved within the function and can be created once with `dispatch` passed in as a parameter:

```js
const debouncedPlanTrip = debounce(({dispatch}) => dispatch(planTrip()), DEBOUNCE_PLAN_TIME_MS)
```